### DISCUSSION TITLE
списки IP адресов

### DISCUSSION THREAD

**Discussion by I-am-all** [remittor/zapret-openwrt#23] [Q&A]
не увидел в инструкции. А как настраиваются обновления списка IP адресов?

**Comment by remittor** [comment]
Автоматическое обновление списков ещё не завезли.
Может и не завезут.

**Reply by DeduskaPikhto** [reply]
Тогда хорошо бы иметь информацию, где эти списки качать, в пригодном формате 
Но вот то, что не завезут, как то не очень. Все же это практически готовое решение, а без обновления списков будет быстро устаревать
